runtime/tests/staking_reward.rs

```diff
@@ -353,6 +353,13 @@ fn elected_validators_receive_transaction_fee_reward() {
 
 			// end era 1, reward payouts are scheduled
 			start_active_era(2);
+
```

Comment:
I don't think this is testing the same thing

Reply:
It's testing the treasury cut the right way. The magic number +6 that was in the previous version is implementation sensitive. You should have some logic that why it must be exacly +6 imbalance during the course of this test.

Reply:
Adding new check is fine but removing old one isn't, this actually documents the behaviour that payouts don't get split equally by the payout process

Reply:
Restored the check.